CMakeLists.txt

```diff
@@ -210,39 +210,36 @@ set(ADMIN_DIR services/user/AdminSys/ui:AdminSys_js)
 set(COMMON_LIB_DIR services/user/CommonSys/common/packages/common-lib/:CommonSysCommonLib_js)
 set(COMMON_LOADER_DIR services/user/CommonSys/common/packages/wasm-loader/:CommonSysLoaderLib_js)
 
-function(add_js_target ENTRY OUTPUT_FILE)
-    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\1 PATH ${ENTRY})
-    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\2 DEP ${ENTRY})
+function(add_js_target ROW OUTPUT_FILE)
+    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\1 PATH ${ROW})
+    string(REGEX REPLACE "^([^:]+):([^:]+)$" \\2 TargetName ${ROW})
+    set(OUTPUT_FILEPATH ${CMAKE_CURRENT_SOURCE_DIR}/${PATH}/dist/${OUTPUT_FILE})
```

Comment:
What did we decide on the merits of hard-coding `/dist` in this?

Reply:
Works for now until we have output files that don't live in a `dist` directory. At which point we can change the parameters to OUTPUT_FILE_SUBPATH and pass in `dist/index.html` etc.